### ISSUE TITLE
Typescript issues

### ISSUE THREAD

**Issue by ash-c** [DeloitteDigitalAPAC/react-habitat-redux#6]
Installed react-habitat-redux 2.0.2 to use with react-habitat 0.6.1 (which we've been using in typescript with no issues), but what's written in the docs doesn't seem to work? There's no ReduxDomFactory to import, so I can't set the factory on the container builder that way.

I can do this:
```
import * as ReactHabitatRedux from "react-habitat-redux";

builder.factory = new ReactHabitatRedux.default.Container(createStore(SelfAssessment)).domFactory();
```

Which compiles, but doesn't actually load on the page as Container is not a constructor. It looks to me like maybe the .d.ts is out of date, but I'm not sure.

I also tried importing the finished .js file, but that had a bunch of compile issues so I didn't go further with that.

**Comment by jennasalau** [comment]
Hi Ash,

Thanks for raising... yes the ts definitions are indeed out of date. I'll get onto this tonight for you.

**Comment by jennasalau** [comment]
This is now resolved in react-habitat-redux 2.0.3 :)

**Comment by ash-c** [comment]
Great, thanks!
